es6-classes/example-class.js

```diff
@@ -0,0 +1,22 @@
+class ExampleClass {
+  constructor(example) {
+    this.example = example;
+  }
```

Comment:
The `ExampleClass` should be empty, a `constructor` is not required.

Reply:
![image](https://user-images.githubusercontent.com/71291742/135501347-50a6e0a0-300b-45e4-9466-6f87e487cc14.png)